packages/twenty-server/src/engine/metadata-modules/field-metadata/utils/compute-column-name.util.ts

```diff
@@ -36,18 +36,37 @@ export function computeColumnName<T extends FieldMetadataType | 'default'>(
 
   return generateName(fieldMetadataOrFieldName.name);
 }
-
-export const computeCompositeColumnName = <
+export function computeCompositeColumnName(
+  fieldName: string,
+  compositeProperty: CompositeProperty,
+): string;
+export function computeCompositeColumnName<
   T extends FieldMetadataType | 'default',
 >(
   fieldMetadata: FieldMetadataInterface<T>,
   compositeProperty: CompositeProperty,
-): string => {
-  if (!isCompositeFieldMetadataType(fieldMetadata.type)) {
+): string;
+export function computeCompositeColumnName<
```

Comment:
nit: I feel like this might get complex over time if this can be used with either FieldMetadata or string which are quite different 🤔

Reply:
It's like that for now since some previous PR, because I'm not yet sure on which constructor I should choose